.github/workflows/typo3_v11ci.yml

```diff
@@ -0,0 +1,36 @@
+# Adapted from https://github.com/TYPO3GmbH/blog/blob/master/.github/workflows/ci.yml
+name: CI
```

Comment:
Can you please add propper naming to differ between TYPO3 11 and 12 testing?

Reply:
Done